docs/meshstack.aws.aad-as-idp.md

```diff
@@ -0,0 +1,80 @@
+---
+id: meshstack.aws.aad-as-idp
+title: AAD as Identity Provider
+---
+## Configure AAD as IdP for AWS SSO
+
+You can trigger everything that is needed for using AAD as an IdP for AWS SSO from within Azure.
+
+### Create an Enterprise Registration
+
+Search for `aws` and choose `AWS Single Sign-on` as the enterprise registration type
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png)
```

Comment:
d: using numbers in the file name make it hard to find the image you are looking for. Naming them by what is content of the image could be more helpful. Especially when you want to add an additonal image at position 4 for example. That would break the whole numbering

Reply:
Good point. That's exactly what I have now with the SSO Enable Picture :-)